code/modules/surgery/organs/augments_arms.dm

```diff
@@ -171,20 +170,20 @@
 
 /obj/item/organ/cyberimp/arm/surgery/emag_act()
 	. = ..()
-	if(locate(/obj/item/kitchen/knife/combat/cyborg) in items_list)
+	if(obj_flags |= EMAGGED)
 		return
 	to_chat(usr, "<span class='notice'>You unlock [src]'s integrated knife!</span>")
 	items_list += new /obj/item/kitchen/knife/combat/cyborg(src)
 	return TRUE
 
 /obj/item/organ/cyberimp/arm/janitor
-	name = "janitor toolset implant"
+	name = "janitorial tools implant"
 	desc = "A set of surgical tools hidden behind a concealed panel on the user's arm."
```

Comment:
The description was the line I pointed out, not the name. It still says "surgical tools" instead of "janitorial tools".